src/js/pages/export/ExportPage.js

```diff
@@ -31,7 +33,19 @@ export class ExportPage extends Component {
     this.props.resetSelected();
   }
 
-  toggleCheck = () => { this.setState({checked: !this.state.checked});}
+  toggleCheck = () => { 
+    this.setState({
+      checked: !this.state.checked,
+      checkedAll: false
+    });
+  }
+
+  toggleCheckAll = () => { 
+    this.setState({
+      checkedAll: !this.state.checkedAll,
+      checked: false
+    });

```

Comment:
`this.setState( prevState => ({checkedAll: !prevState.checkedAll, checked: false})`